assets/scripts/src/choices.js

```diff
@@ -485,98 +486,103 @@ class Choices {
-            let notice;
+              // Otherwise show a notice
+              let dropdownItem;
+              let notice;
 
-            if (this.isSearching) {
-              notice = isType('Function', this.config.noResultsText) ?
-                this.config.noResultsText() :
-                this.config.noResultsText;
+              if (this.isSearching) {
+                notice = isType('Function', this.config.noResultsText) ?
+                  this.config.noResultsText() :
+                  this.config.noResultsText;
 
-              dropdownItem = this._getTemplate('notice', notice, 'no-results');
-            } else {
-              notice = isType('Function', this.config.noChoicesText) ?
-                this.config.noChoicesText() :
-                this.config.noChoicesText;
+                dropdownItem = this._getTemplate('notice', notice, 'no-results');
+              } else {
+                notice = isType('Function', this.config.noChoicesText) ?
+                  this.config.noChoicesText() :
+                  this.config.noChoicesText;
 
-              dropdownItem = this._getTemplate('notice', notice, 'no-choices');
-            }
+                dropdownItem = this._getTemplate('notice', notice, 'no-choices');
+              }
 
-            this.choiceList.appendChild(dropdownItem);
+              this.choiceList.appendChild(dropdownItem);
+            }
           }
         }
-      }
 
-      // Items
-      if (this.currentState.items !== this.prevState.items) {
-        // Get active items (items that can be selected)
-        const activeItems = this.store.getItemsFilteredByActive();
+        // Items
+        if (this.currentState.items !== this.prevState.items) {
+          // Get active items (items that can be selected)
+          const activeItems = this.store.getItemsFilteredByActive();
 
-        // Clear list
-        this.itemList.innerHTML = '';
+          // Clear list
+          this.itemList.innerHTML = '';
 
-        if (activeItems && activeItems) {
-          // Create a fragment to store our list items
-          // (so we don't have to update the DOM for each item)
-          const itemListFragment = this.renderItems(activeItems);
+          if (activeItems && activeItems) {
+            // Create a fragment to store our list items
+            // (so we don't have to update the DOM for each item)
+            const itemListFragment = this.renderItems(activeItems);
 
-          // If we have items to add
-          if (itemListFragment.childNodes) {
-            // Update list
-            this.itemList.appendChild(itemListFragment);
+            // If we have items to add
+            if (itemListFragment.childNodes) {
+              // Update list
+              this.itemList.appendChild(itemListFragment);
+            }
           }
         }
-      }
 
-      this.prevState = this.currentState;
+        this.prevState = this.currentState;
+        this._storeNeedsRendering = false;
```

Comment:
In the RWC, this line `_storeNeedsRendering = false` appears **after** the curly-brace on line 583.

Reply:
Good catch, I'll make the change, it should be after the curly-brace